### PR TITLE
Env variables

### PR DESCRIPTION
**Pull request by aldjonz**

### CODE REVIEW

File: src/components/Map/SearchPane/SearchBar/AutoSuggestEntities/AutoSuggestEntities.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ const AutoSuggestEntities: React.FC<PageProps> = (props) => {
     const handleInputFocus = async() => {
       props.setIsSearching(true);
 
-      const response = await fetch('https://raw.githubusercontent.com/aldjonz/ccm-json/main/entities.json');
+      const response = await fetch(`${process.env.REACT_APP_DATABASE_URL}aldjonz/ccm-json/main/entities.json`);
       const data = await response.json();
```
Comment (croesus) [inline src/components/Map/SearchPane/SearchBar/AutoSuggestEntities/AutoSuggestEntities.tsx:117]: I suggest leaving the trailing slash off the .env URL like you have for `REACT_APP_BASE_URL` - it reads a bit better . Also I think the `REACT_APP_DATABASE_URL` should include `/aldjonz/ccm-json/main` - this isn't going to be the path on the live version

**Review by croesus — COMMENTED** [review]
A bit naughty combining the import paths change with the .env change as they're unrelated. Would have been better in two branches so they could be rolled back independently if needed. But we'll let it lie :)

File: src/service/fetchURL/individualEntity/fetchIndividualEntity.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export const fetchIndividualEntity:any = async(entityID:string) => {
     try {
-        const response = await fetch(`${process.env.REACT_APP_DATABASE_URL}carbonmap/ccm-front-end/master/dummy_data/${entityID}`);
+        const response = await fetch(`${process.env.REACT_APP_DATABASE_URL}${entityID}`);
```
Comment (croesus) [inline src/service/fetchURL/individualEntity/fetchIndividualEntity.ts:3]: Should this have a "/" between `REACT_APP_DATABASE_URL` and `entityID` ?

Comment (aldjonz) [reply src/service/fetchURL/individualEntity/fetchIndividualEntity.ts:3]: No, entityID should be passed in with "/" at the begging of the string

**Pull request merged**